react-native/metro.config.js

```diff
@@ -30,6 +30,13 @@ module.exports = {
         };
       }
 
+      if (moduleName.startsWith('electric-sql/config')) {
```

Comment:
Can this be wildcarded in some way?

Reply:
I.e.: we have three imports, same pattern, can we make a more dynamic filter than matches any electric-sql import?

Reply:
`filePath` cannot be inferred from matching pattern and if we capture all `electric-sql/` then we also have to implement the default case. I think it is better to keep this flat and simple.